scoring.js

```diff
@@ -0,0 +1,83 @@
+
+
+function passingPoints(player) {
+    let ptsPeryard = player.stats.passing.yards / 25
+    let ptsPertouch = player.stats.passing.touchdowns * 6
+    let ptsPerinter = player.stats.passing.interceptions * -3
+    return ptsPeryard + ptsPertouch + ptsPerinter
+}
+
+function receivingPoints(player) {
+
+    let ptsPerrecp = player.stats.receiving.receptions
```

Comment:
These should all be `const`, you don't ever reassign them a new value. Basic rule, use `const` until something breaks, then switch it to `let`